scripts/release.py

```diff
@@ -908,6 +916,12 @@ def main():
                           default='origin',
                           help="""The Git alias of remote URL.
 Default: '%default'""")
+        parser.add_option('--depth', action="store", type="int", dest='depth', default=None,
+                          help="""Create a shallow clone with a history truncated to the specified number of commits. 
+Default: '%default'""")
+        parser.add_option('--unshallow', action="store_true", dest='unshallow', default=False,
+                          help="""If the source repository is shallow, fetch as much as possible so that the current 
+repository has the same history as the source repository. Default: '%default'""")
```

Comment:
and if the source is complete (ie not shallow), it will convert the shallow repository to a complete one, removing all the limitations imposed by shallow repositories ?

Reply:
yep, also there aren't much limitations from git 1.9 as we can commit & push (that's why I added links to the docs)